apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -50,77 +52,133 @@ export function InstructorCourseAdminInstances({
+                                      output: row.sync_errors,
+                                    })
+                                  : row.sync_warnings
+                                    ? SyncProblemButton({
+                                        type: 'warning',
+                                        output: row.sync_warnings,
+                                      })
+                                    : ''}
+                                <a
+                                  href="${resLocals.plainUrlPrefix}/course_instance/${row.id}/instructor/instance_admin"
+                                  >${row.long_name}</a
+                                >
+                              </td>
+                              <td class="align-left">${row.short_name}</td>
+                              <td class="align-left">${row.formatted_start_date}</td>
+                              <td class="align-left">${row.formatted_end_date}</td>
+                              <td class="align-middle">${row.enrollment_count}</td>
+                            </tr>
+                          `;
+                        })}
+                      </tbody>
+                    </table>
+                  </div>
+                `
+              : html`
+                  <div class="my-4 card-body text-center" style="text-wrap: balance;">
+                    <p class="font-weight-bold">No course instances found.</p>
+                    <p class="mb-0">
+                      A course instance is a single offering of a course, such as "Fall 2024" or
+                      "Fall 2024, Section M".
+                    </p>
+                    <p>
+                      Learn more in the
+                      <a
+                        href="https://prairielearn.readthedocs.io/en/latest/courseInstance/"
+                        target="_blank"
+                        >course instance documentation</a
+                      >.
+                    </p>
+                    ${run(() => {
+                      if (resLocals.course.example_course) {
+                        return html`<p>You can't add course instances to the example course.</p>`;
+                      }
+                      if (!resLocals.authz_data.has_course_permission_edit) {
+                        return html`<p>Course Editors can create new course instances.</p>`;
+                      }
+                      if (resLocals.needToSync) {
+                        return html`
+                          <p>
+                            You must
+                            <a href="${resLocals.urlPrefix}/course_admin/syncs">sync the course</a>
+                            before creating a new course instance.
+                          </p>
+                        `;
+                      }
+                      return html`
+                        <form class="ml-auto" name="add-course-instance-form" method="POST">
```

Comment:
Nit (`ml-auto` doesn't do anything here, and I don't think this form needs a name):

```suggestion
                        <form method="POST">
```